README.md

```diff
@@ -448,11 +449,41 @@ type ReadonlyNestedProps = DeepReadonly<NestedProps>;
 
 ---
 
+### `DeepRequired<T>`
+
+Required that works for deeply nested structures
+
+**Usage:**
+
+```ts
+import { DeepRequired } from 'utility-types';
+
+type NestedProps = {
+  first?: {
+    second?: {
+      name?: string | null | undefined;
+    };
+  };
+};
+type RequiredNestedProps = DeepRequired<NestedProps>;
+// Expect: {
+//   first: {
+//     second: {
+//       name: string;
+//     };
+//   };
+// }
+```
+
+[⇧ back to top](#mapped-types)
+
+---
+
 ## Flow's Utility Types
 
 ### `$Keys<T>`
 
-get the union type of all the keys in an object type `T`  
```

Comment:
Please bring back all trimmed line-ends, notice there is more below...

Reply:
Reverted.